string name
bool is_target
geometry_msgs/Vector3 position
